_docs/setup/verify.md

```diff
@@ -31,31 +31,29 @@ folder within your downloads directory, do so.
 6. If you have used Glacier before, *and* you know you have the Glacier public key
 imported into a local GPG keyring, skip the next step. (If you don't know,
 that's fine; proceed as normal.)
-7. Obtain the Glacier "public key," used to cryptographically verify the
-  <a href="#" class="popovers" data-toggle="popover" data-placement="top" title=""
-  data-content="
-  Technical details: Glacier's GPG keys are handled with good security practices. They were generated while booting off an Ubuntu Live USB on a factory-new laptop with the wireless card removed, and transferred via USB to a MacBook. The private key is not stored in the cloud. The public key is hosted separately from our software distributions, on Keybase, secured with separate credentials (all of which are in password managers).
-  ">protocol document</a>.
-
+5. Obtain the Glacier PGP public key, used to cryptographically verify the protocol document.
+
+    Technical detail: Glacier's GPG keys are handled with good security practices.
+    They were generated while booting from an Ubuntu Live USB on a factory-new laptop
+    with the wireless card removed, and transferred via USB to a MacBook. The private
+    key is not stored in the cloud. The public key is hosted separately from the
+    software distributions, on [Keybase](https://keybase.io/), and secured with separate credentials stored
+    in password managers.
    **If you are ever using Glacier in the future and notice that this step has
    changed (or that this warning has been removed), there is a security risk.**
    Stop and
-   <a href="#" class="popovers" data-toggle="popover" data-placement="top" title=""
```

Comment:
Good catch; these popovers are invisible in the current PDF, which is obviously unacceptable.